os/binfmt/libelf/libelf.h

```diff
@@ -464,10 +464,12 @@ int elf_cache_read(int filfd, uint16_t binary_header_size, FAR uint8_t *buffer,
 struct bin_addr_info_s {
 	struct bin_addr_info_s *flink;
 	char bin_name[BIN_NAME_MAX];
-	uint32_t text_addr;
-	uint32_t rodata_addr;
-	uint32_t data_addr;
-	uint32_t bss_addr;
+	uint32_t text_paddr;
+#ifdef CONFIG_OPTIMIZE_APP_RELOAD_TIME
```

Comment:
How about changing the config name to CONFIG_DYNAMIC_RELOAD?

Reply:
or CONFIG_DYNAMIC_RELOAD_APP ?

Reply:
This is the config for optimization when reloading. There is already another config which related to dynamic loading.

Reply:
Config is already in use, so no need to change it.